### ISSUE TITLE
System-wide tag

### ISSUE THREAD

**Issue by ericvoit** [thomas-fossati/draft-ear#13]
We need a predefined/standardized system-wide tag so that a Relying Party know the appraisal spans the whole system.  When where there are submods, we need to talk about the relationship between the system-wide appraisals, and the appraisals for any actual subsystems.

**Comment by thomas-fossati** [comment]
The "multiple submods" case materialises only when the appraisal is not complete.

An RP is not supposed to consume directly this kinds of EAR -- unless it's coupled with the Attester, which should be the exception in an AR4SI world.  In the "uncoupled" case, RP will need to forward this partial EAR to some other Verifier to complete the appraisal.

A complete appraisal will typically only have one submod, which has system-wide scope.

**Comment by ericvoit** [comment]
I don't think it is decided that a complete appraisal will summarize all the elements of a system.  At least with routers, there are different levels of system distribution.  And you might have each line card requiring its own appraisal each has a TPM.  I can imagine there are server clusters which also will need to provide a distributed view.

We could update the intro to the EAR document to show that such aggregate attestations are out of scope.  And that would also be a way to close the issue.
